docs/user-manual/en/address-model.md

```diff
@@ -720,6 +721,12 @@ before it will begin to dispatch messages. Default is `-1` (wait forever).
 closed on a queue before redistributing any messages. Read more about
 [clusters](clusters.md#message-redistribution).
 
+`clustered-queues` enables cluster-wide creation of `ANYCAST` queues. 
```

Comment:
why only anycast, surely queue type should not matter, and use cases for mutlicast would exist, e.g. share durable subscriptions